### DISCUSSION TITLE
Awgb verbied direct en indirect onderscheid

### DISCUSSION THREAD

**Discussion by floort** [MinBZK/Algoritmekader#32] [Ideas]
In de uitleg over non-discriminatie staat uitleg over een publicatie van het College voor de Rechten van de Mens.
https://github.com/MinBZK/Algoritmekader/blob/main/docs/bouwblokken/fundamentele%20rechten/non-discriminatie.md

In deze samenvatting lees ik een interpretatie die volgens mij volgens de Algemene wet gelijke behandeling teveel ruimte overlaat.
De Awgb verbied (onder omstandigheden en met uitzonderingen) namelijk direct en indirect onderscheid en dat gaat veel verder dan risicoprofielen uitsluitend richten op of uitsluitend gebaseerd zijn op een beschermde categorie.

De formulering van Artikel 1, lid 1.c maakt volgens mij de relevantie voor algoritmes wel duidelijk:
> indirect onderscheid: indien een ogenschijnlijk neutrale bepaling, maatstaf of handelwijze personen met een bepaalde godsdienst, levensovertuiging, politieke gezindheid, ras, geslacht, nationaliteit, hetero- of homoseksuele gerichtheid of burgerlijke staat in vergelijking met andere personen bijzonder treft.

Omdat pull request #21 wat langer blijft hangen dan andere pull requests is het mij niet duidelijk of het de bedoeling is om verwijzingen naar de Awgb op te nemen en wat de juiste vorm is om deze wetgeving mee te nemen in andere verbetersuggesties.

Er zijn denk ik (minimaal) drie belangrijke vragen die meekomen met de combinatie van de Awgb en AVG die belangrijk zijn voor het algoritmekader:

1. Moet het gebruik van een algoritme dat direct of indirect onderscheid maakt op basis van de beschermde categorieën getoetst worden aan de Awgb?
2. Zijn persoonsgegevens die direct of indirect betrekking hebben op de beschermde categorieën uit de Awgb, maar niet gebruikt worden om onderscheid te maken in het algoritme wel relevant voor de toepassing van het algoritme en is de verwerking daarmee rechtmatig (zie artikel 5(1)(c) AVG)?
3. Is er behoefte aan advies over het toetsen of input data (vooral indirect) betrekking heeft op de categorieën uit de Awgb?

**Comment by jaspervanderheide** [comment]
Ha Floor. Je PR blijft even hangen omdat een collega op vakantie is. Zij heeft dat overzicht gemaakt, dus zij kan beste beoordelen of het erbij hoort. (Wat mij betreft hoort het erbij als ik de lijst zie overigens.)

CvRM gaat in hun stuk overigens wel uitgebreid in op indirecte discriminatie, maar niet in de Richtlijnen. Dat moet vanzelfsprekend wel duidelijk in het uiteindelijke kader komen.

Om concreet je vragen te beantwoorden:

1. Ja, lijkt me wel. Eigenlijk wil je elk algoritme op een (proportionele) manier toetsen, omdat indirect onderscheid niet altijd gemakkelijk te zien is.
2. Dit is een goede vraag voor @IsabelBarbera's werkgroep.
3. Ja! Wij zijn nu een werkgroep non-discriminatie aan het opstarten, dus je hulp is welkom! Dat kan ook gewoon via Github. We zijn nu nog wat in de verkennende fase, dus deze discussie helpt al enorm. Verder dus zoeken naar rapporten, bestaande tools en nuttige definities. Maar straks willen we ook kijken hoe we dat toetsen praktisch kunnen maken.

Afsluitend nog een ping voor @ruthkoole

**Comment by floort** [comment]
Dankjewel. Ik wil geen druk zetten om de PR sneller te behandelen, maar zag een mogelijke blokkade voor verdere voorstellen op dit punt. Die is er dus niet. 

Voor punt 1: je zegt op een 'proportionele' manier toetsen. Dat klinkt redelijk, maar ik weet niet wat het betekent. Ik denk dat het beter is om vast te houden aan artikel 5(2) AVG en te zeggen dat de verwerkingsverantwoordelijke in staat moet zijn om o.a. de rechtmatigheid aan te tonen als er persoonsgegevens worden verwerkt. Dat omvat uiteindelijk ook rechtmatigheid onder de Awgb. Ik denk niet dat het handig is om de indruk te wekken dat er niet aan 5(2) AVG voldaan hoeft te worden als dat teveel werk wordt gevonden.

**Reply by jaspervanderheide** [reply]
De Awgb moet inderdaad sowieso gebruikt worden. Even aan mijn collega hoe die op die lijst moet.

Eens dat we aan de wettelijke vereisten moeten voldoen, de indruk wil ik ook niet wekken dat we dat niet gaan doen. Waar wij nog wel mee puzzelen is _hoe_ je dat aantoont. Is dat hetzelfde in elke situatie? Zijn er lichtere en zwaardere vormen die in verschillende situaties geschikt zijn om te voldoen aan die eis? Dat sluit eigenlijk aan bij jouw derde vraag en ideeën zijn welkom. Het is dus ook mogelijk dat er geen verschil is in verschillende situaties.

**Reply by floort** [reply]
Dat het zo lastig is is precies de reden dat ik er over begin. Volgens mij is het goed te doen om enkele voorbeelden te geven van hoe je zoiets kan aanpakken, inclusief eventuele lichtere vormen en relevante overwegingen voor als het niet haalbaar is om rechtmatigheid aan te tonen.

**Reply by jaspervanderheide** [reply]
Precies, dat willen we inventariseren! Twee concrete opties zijn een statistische toets met bijzondere persoonsgegevens gekoppeld aan de uitkomsten van een algoritmes of het gebruik van de [bias scan van Algorithm Audit](https://algorithmaudit.eu/nl/bias_scan/). Wat mij betreft maken we lijstjes van mogelijke methoden op de pagina. Dan kunnen we daarna selecteren wat we wel of niet opnemen en wanneer.

**Reply by floort** [reply]
Ik denk dat als je de Awgb plus 5(1)(c) AVG samen hanteert dat je een dergelijke statistische toets op de input van het algoritme moet uitvoeren. Door alleen aan de output te toetsen neem je 5(1)(c) AVG niet mee en kan het zijn dat je een vergelijkbare toets alsnog op de input moet uitvoeren. 

Heel concreet: als je een van de beschermde categoriën (direct of indirect) wel als input gebruikt, maar uit de statistische toets blijkt dat de output er niet van afhankelijk is, dan had je dat stuk van de input ook niet nodig. En een verwerking van persoonsgegevens die niet nodig is voor het doel is onrechtmatig onder 5(1)(c) AVG.

**Reply by jaspervanderheide** [reply]
Helder, ik had het nog niet zo bekeken. Ook omdat je dan op het terrein van gegevensbescherming begeeft. Hoe zie jij dat tussen onze werkgroepen @IsabelBarbera?

Het zijn wel twee verschillende toetsen als ik je goed begrijp. Op input is de vraag: in hoeverre heeft het meenemen van deze gegevens effect op het de effectiviteit van het algoritme. Bij output is vooral mijn/de vraag: leidt het algoritme tot onwenselijke/discriminerende verschillen tussen beschermde categorieën. Als je alleen naar input kijkt, dan mis je dat. Ook omdat als het goed is de input zo weinig mogelijk beschermde categorieen bevat (hopelijk), maar dit wel bij de output tot indirecte discriminatie leidt.

Dus in die zin zou mijn lezing zijn dat de Awgb primair baat heeft bij de output kant.

**Reply by floort** [reply]
Ik lees de Awgb net iets anders. Het gaat niet direct over mogelijke onwenselijke verschillen. De eerste toetst is of je onderscheid maakt op basis van specifieke eigenschappen (direct of indirect). Dat is vrij mechanisch: komt het algoritme met een andere uitkomst als je die eigenschap veranderd? Gaat er direct of indirect informatie in die betrekking heeft op die beschermde eigenschappen? Zo nee, ben je klaar met de Awgb want die is niet van toepassing. Zo ja, moet je verder toetsen. Is die informatie relevant voor de beslissing van het algoritme? Zo nee, dan heb je de informatie überhaupt niet nodig. Zo ja, kom je aan allerlei afwegingen onder de Awgb toe om te kijken of het onderscheid rechtmatig is. 

Overigens zou ik helemaal in het begin beginnen met de vraag of het een context is waar de Awgb van toepassing is (arbeid, sociale zekerheid, ...). Dan valt het meeste al af namelijk. Dat is niet de inhoudelijke vraag die hier relevant is, maar de impact van het moeten aantonen dat je niet onrechtmatig discrimineert is dit relatief beperkt.

**Reply by jaspervanderheide** [reply]
Hmmm, maar hier ga je er wel vanuit dat je vooraf weet dat een eigenschap gerelateerd is aan een bijzondere categorie. Ik ken de details niet, maar het algoritme mbt uitwonendebeurs van DUO lijkt een voorbeeld van waar dat niet evident is vooraf.

**Reply by floort** [reply]
Heb je een link naar ergens waar ik meer kan lezen over dat algoritme? Ik kan het niet vinden in het algoritmeregister.

**Reply by jaspervanderheide** [reply]
En ik ben geen jurist in alle eerlijkheid, maar de definitie van "indirect onderscheid" doet mij vermoeden dat het hier wel om resultaat gaat?

Voor DUO: https://www.groene.nl/artikel/ik-dacht-gewoon-ik-pak-je. Er zijn ook wat Kamerdocumenten waar wat meer instaat.

**Reply by floort** [reply]
Ik lees onderscheid als: is het van invloed op een beslissing? Wat de gevolgen zijn van die beslissing is relevant, maar wel een andere vraag.

En in het artikel lees ik speculatie over dit punt.

> Controleert Duo studenten met een migratieachtergrond dan vaker? Volgens Duo zit etniciteit of nationaliteit niet in het algoritme. Maar het kan ook indirect zijn, legt Lucas Haitsma uit, die promoveert op algoritmen en discriminatie bij overheidsorganisaties met een handhavingsfunctie. Als je mbo als risicofactor gebruikt of een familierelatie zwaar weegt, dan kun je daarmee ook een bepaalde groep targeten.

Maar het algoritme zelf is niet transparant dus is het lastig als voorbeeld te gebruiken. Dit is wel waarom ik bij het algoritmeregister heb gepleit voor meer interessante/lastige registraties én velden die ingaan op dit punt. Ook registraties in het register bevatten bijna nooit voldoende informatie om dit soort vragen te beantwoorden of om te controleren of die vragen überhaupt zorgvuldig zijn beantwoord.

**Reply by jaspervanderheide** [reply]
Bij direct onderscheid ben ik het met je eens, maar bij indirect onderscheid moet je eerst bepalen of het die groepen in verschillende mate treft. Dat is een output vraag. Immers kan ik met vijf ogenschijnlijk objectieve criteria toch discrimineren. Dat moet je onderzoeken aan achterkant. Je kan wel de individuele criteria controleren op correlatie, maar dan mis je de samenhang.

Ik zal nog even zoeken, want er zijn wel openbare bronnen die wat zeggen over de rekenregels van DUO.

**Reply by floort** [reply]
Daar ben ik het mee eens. Maar als de output niet gevoelig is voor de Awgb criteria moet je alsnog de input testen vanwege de AVG. Andersom als de input zelfs geen correlatie heeft met de Awgb criteria dan hoef je de output daar niet ook op te testen. En ik vermoed dat het testen van de input eenvoudiger is dan het testen van de output, maar dat weet ik niet zeker.

**Reply by jaspervanderheide** [reply]
Met het eerste ben ik het eens. Echter wil je de input dan wel in samenhang controleren op correlatie, niet alleen de losse criteria. En dan zit je al vrij snel op het toetsen van output. Ook zie ik niet direct hoe input testen gemakkelijker is, want statistisch zijn er voor beide voldoende hulpmiddelen. Echter hebben organisaties veelal niet de beschermde gegevens om die correlatie te toetsen, zowel op input als output. Zon bias scan van Algorithm Audit kan daar een oplossing bieden.

**Reply by jaspervanderheide** [reply]
Heb je misschien zelf een idee hoe je die input data zou willen controleren, wat de stappen zijn.

**Reply by floort** [reply]
Dat hoopte ik gedeeltelijk hier samen uit te werken. Het belangrijkste is volgens mij om goed uit te leggen dat er een verplichting is om aan te kunnen tonen dat de verwerking rechtmatig is, inclusief onder de Awgb. Hoe je dat kan onderbouwen is context-afhankelijk, maar er kunnen wel een aantal tips gegeven worden.
Ik hoop eigenlijk op een equivalent van deze handleiding van het CBS, maar dan gericht op deze context: https://www.cbs.nl/nl-nl/onze-diensten/methoden/statistische-methoden/output/statistische-beveiliging
Daar zou bijvoorbeeld een kort lijstje met handvaten uit kunnen komen zoals: Zorg dat van alle eigenschappen in de input data een specifieke beschermde categorie niet voor meer dan 90% aanwezig is. Voorbeeld: in een bible-belt dorp kan woonplaats een sterke proxy zijn voor religie.

**Reply by jaspervanderheide** [reply]
Dan ben je aan goede adres, maar ik dacht dat je misschien al concrete aaanpak had. Had je overigens de handreiking non-discriminatie by design al gezien?

**Reply by IsabelBarbera** [reply]
Naast correlatie, heb je nu ook andere methodieken zoals causal feature selection voor de samenhang tussen indicatoren.

Deze discussie lijkt me super interessant voor een fysieke brainstormsessie! We kunnen samen de samenhang tussen wet en techniek bekijken om de normen aan de juiste processen te koppelen. @floort,  wanneer kom je op bezoek? 😊

**Reply by floort** [reply]
Vanwege het enorm bede spectrum aan soorten algoritmes en toepassingsgebieden vind ik het heel lastig om eenduidige methoden te noemen. 
Ik kan me heel goed een statistische toets voorstellen waaruit duidelijk wordt dat een veld "postcode" in een database in sommige gevallen een hele sterkte correlatie heeft met bijvoorbeeld religie. Maar hoe doe je dat met camera-beelden? Dan moet je waarschijnlijk hele andere methoden gebruiken.
Wat ik wil voorkomen is dat de vraag uit de weg wordt gegaan omdat de uitvoering lastig is. Het is ook goed mogelijk dat de juiste conclusie is dat voor sommige systemen we niet weten hoe die aantoonbaar compliant ingezet kunnen worden en dat dus de inzet niet rechtmatig is totdat iemand de rechtmatigheid wel kan aantonen. 

Volgens mij zou dat een goede basis zijn voor een brainstorm over concrete maatregelen. Aanvullend zouden registraties in het algoritmeregister met goede voorbeelden en voldoende detail om geinformeerd als voorbeeld te kunnen gebruiken ook helpen. Is dat haalbaar?

**Reply by jaspervanderheide** [reply]
Het aanleveren van registraties voor het register is aan overheidsorganisaties zelf. Daar hebben wij vanuit projectteam helaas weinig invloed op. Dus wanneer er goede voorbeelden zijn (die aan jouw criteria voldoen), weet ik niet.

Overigens eens met je analyse dat het per algoritme en toepassingsgebied kan verschillen. Daar moeten we onderscheid in gaan maken.

**Reply by floort** [reply]
Het is duidelijk dat jullie niet over de inhoud van de registraties gaan. Ik denk dat goede voorbeelden helpen en dat het kan lonen als instellingen die misschien worstelen met registraties omdat niet helemaal bekend is hoe ze moeten uitleggen waaraan ze voldoen nuttig kunnen zijn als voorbeeld gebruikt worden. Als ze niet al in het register zitten kan dat ook op een andere manier. Het gaat mij daarbij niet om mijn criteria (geen idee wat die zijn), maar om zo nuttig mogelijk voorbeelden die zo goed mogelijk uitgewerkt zijn.

**Reply by jaspervanderheide** [reply]
Er loopt in ieder geval een toegezegd onderzoek naar de DUO-casus die ik eerder noemde. Ik ken de inhoud daarvan niet, maar dat geeft na publicatie hopelijk voldoende diepgang voor discussie. Anders zal ik kijken of er voor deze specifieke discussie casussen zijn.

**Reply by floort** [reply]
Ik heb bij deze discussie een paar voorbeelden genoemd: https://algoritmes.pleio.nl/groups/view/6719c09d-3b0d-4b77-b26f-578fe6cf1fc8/applicatie-algoritmeregister/discussion/view/3f7ad813-f33a-4bac-adad-9b0c4b9a708d/verzameling-concrete-incidenten

Behalve DUO is het RISC voorbeeld wel een interessante maar lastig te bespreken vanwege een gebrek aan informatie. Een spamfilter casus is misschien een veilige optie om te bespreken omdat praktisch iedereen dat heeft en er conceptueel een paar interessante haakjes aan zitten. Hoe bepaal je bijvoorbeeld of discriminitie op basis van IP ranges wel of geen indirect onderscheid is uit de Awgb?

Veel animo om het op pleio te bespreken is er niet helaas. Misschien eens kijken of de uitnodiging van @IsabelBarbera help om dit soort praktische casussen wel bespreekbaar te maken?

**Comment by Mildoooo** [comment]
De handreiking  non-discriminatie stelt dat het non-discriminatierecht uitgangspunten geeft en geen ge- of verboden. Het recht geeft een standaard, een uitgangspunt of een principe, maar daarop bestaan altijd 
uitzonderingen. De context speelt een belangrijke rol. Discriminatie (direct/indirect) op basis van directe gronden zoals ras, godsdienst, leeftijd zal weliswaar in veel gevallen verboden zijn, maar je zult altijd de vragen  moeten aflopen of een specifieke geval tot benadeling zal leiden. Vervolgens moet een keuze te rechtvaardigen zijn. Is er een goede reden voor het gemaakte onderscheid? 1) is er een wettelijke uitzondering  En 2) is er een objectieve rechtvaardiging en is er een legitiem doen. Is het middel dat ik gebruik om mijn doel te bereiken passend, noodzakelijk en evenredig? 
Op pagina 10 staat een mooi schema. https://open.overheid.nl/documenten/ronl-3f9fa69c-acf4-444d-96e1-5c48df00eb3c/pdf 
Algoprudentie kan een waardevolle bijdrage leveren aan normontwikkeling. Vooral in gevallen waarbij het niet zo evident is dat er sprake is van een verbod.

**Reply by floort** [reply]
Ik snap de nuancering daarin: onderscheid is onder omstandigheden toegestaan zoals ook in de Awgb staat beschreven. Ik benader het vanuit artikel 5(2) AVG waarbij moet kunnen worden aangetoond dat de verwerking rechtmatig is (5(1)(a) AVG) en daarmee ook moet kunnen worden aangetoond dat, waar relevant, aan de Awgb wordt voldaan. 

Het is niet mijn intentie om te stellen dat onder de Awgb het verbod op discriminatie absoluut is (dat is het niet). Maar je zal in de concrete situatie moeten kunnen aantonen dat de Awgb niet van toepassing is, of dat je onder een uitzondering van het discriminatieverbod valt. De nuance dat het verbod niet absoluut is zat verstopt in punt 1 van mijn post hier bovenaan. Niet expliciet benoemd, maar niet vergeten.
